tooling/github-strategy.md

```diff
@@ -0,0 +1,92 @@
+| What Repositories are the Perspective’s Focus?​ | Sensitive repositories: Those whose release has market impact or is core infrastructure with safety implications.​ | Highly reusable packages, internal reusable tools, templates, examples, or code that builds services many people have stake in like websites, SDKs, etc. ​ |
+
+The opportunity to balance both needs comes from observation that there is not much overlap between the repositories that each perspective cares about the most.
+
+## Visibility Differences by InnerSource Project Type
+
+There are several types of InnerSource projects, each with its own characteristics and considerations. The table provides an overview of these project types and their visibility differences within an organization. What kinds of InnerSource seeds does your organization have? What areas in your organization would you like to start with when strategically adopting InnerSource? Use this categorization to help you categorize your project.
+
+| Type of InnerSource​ | InnerSource Participation method​ | Significant Communication before pull request or reuse?​ | Will owning party likely know non-owning party beforehand​ | numbers of chances for InnerSource participation killed off by org-level visibility boundaries​ | Enterprise impact of InnerSource action not occurring​ | Likely sensitivity of the code if maliciously released​ |
+| --- | --- | --- | --- | --- | --- | --- |
+| Learning, copy and pasting (examples, templates)​ | Learning​ | None​ | No​ | Most​ | Entropy and lost time.​ | Probably low​ |
+| Use Reusable tools built for Company circumstances. ​ | Reuse​ | None-little​ | No​ | Many​ | Loss of efficiency and speed. ​ | Probably low​ |
+| Content change, fix, add, or update (websites, documentation)​ | Contribution, small ones​ | None-little​ | No​ | Many​ | Wasted time due to bad information.​ | Probably low​ |
+| Build within someone else's internal service | Contribution, within well-defined parameters​ | Yes​ | No​ | Some​ | More things never done at all.​ | Probably low but more exceptions.​ |
+| Deduplication. Don't built same thing twice, build general solution.​ | Contribution ​ | Yes​ | Varies​ | Some​ | General purpose code written less often. ​ | Depends, some very high.​ |
+| Ensure alignment between related projects​ | Contribution ​ | Yes​ | Likely​ | A few​ | Loss of efficiency and speed.​ | Depends, some very high.​ |
+| Not get slowed down by who is owning ​ | Contribution ​ | Yes​ | Likely​ | A few​ | Loss of efficiency and speed.​ | Depends, some very high.​ |
+
+## Project Participation Difficulty Level by Setting
+
+The difficulty level to join a project depends on your access rights to GitHub Enterprise and the repository type. It should be noted that GitHub settings in the enterprise often use SCIM and work with IdPs such as Microsoft Entra and Okta, but the participation hurdle can also vary depending on the settings on the IdP side.
```

Comment:
From @dellagustin  ([The original review comment](https://docs.google.com/document/d/1lPR90dIiWKlsNlV46xhGkR9pP0GNYmeAB7xOk-_MAYo/edit?disco=AAAA1q2rBb8))

General comment, at first occurrence of the term, I recommend using the full term and showing the acronym, e.g. System for Cross-domain Identity Management (SCIM).